django_walletpass/classviews.py

```diff
@@ -27,6 +27,10 @@ def get_pass(pass_type_id, serial_number):
     return get_object_or_404(Pass, pass_type_identifier=pass_type_id, serial_number=serial_number)
 
 
+def web_service_root_view(request):
```

Comment:
this is not a classview. should either inherit from `View` or in views.py I assume. @cherijs

Reply:
However, it seems that this view should be taken out. 

I understood when this error came about not being able to open web_service_url:
`WALLETPASS["SERVICE_URL"] = 'https://xxx/api/passes/'`

`[2023-07-09 12:18:47 +0300] Register task (for device 6d07103565a61da35b0cb6047736518a, pass type pass.id.lolo.tickets, serial number R1e7lciJx5K_j6_m4iNouABbUXY; with web service url https://xxx/api/passes/) encountered error: Unexpected response code 404`

but i was wrong...